sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -74,7 +74,6 @@ private[hive] class SparkExecuteStatementOperation(
     // RDDs will be cleaned automatically upon garbage collection.
     logDebug(s"CLOSING $statementId")
     cleanup(OperationState.CLOSED)
-    sqlContext.sparkContext.clearJobGroup()
```

Comment:
Nit: make that logDebug `logInfo(s"Closed statement with $statementId")`, to make it consistent with other logs.

Reply:
> Nit: make that logDebug `logInfo(s"Closed statement with $statementId")`, to make it consistent with other logs.

maybe `logInfo(s"Close statement with $statementId")` ?

Reply:
I'm fine with both Close and Closed. I see that you already committed Closed.